python/pacemaker/_cts/cib.py

```diff
@@ -382,43 +379,15 @@ def log(self, args):
     def debug(self, args):
         self.CM.debug("cib: %s" % args)
 
-    def register(self, methodName, constructor, *args, **kargs):
-        """register a constructor"""
-        _args = [constructor]
-        _args.extend(args)
-        setattr(self, methodName, ConfigFactoryItem(*_args, **kargs))
-
-    def unregister(self, methodName):
-        """unregister a constructor"""
-        delattr(self, methodName)
-
-    def createConfig(self, name="pacemaker-1.0"):
-        if name == "pacemaker-1.0":
-            name = "pacemaker10"
```

Comment:
I don't see any problem with dropping 1.0, but why?

Reply:
If you look at [a71f29c](https://github.com/ClusterLabs/pacemaker/pull/3150/commits/a71f29cc28eeeebe202b3d04f660736a1b563372), you'll see there's not actually support for 1.0.  It registers attributes for pacemaker12, pacemaker20, and pacemaker30.  Even though the default is pacemaker-1.0 which maps to the pacemaker10 attribute, there's nothign registered for that.  It'll just end up defaulting to 3.0 anyway.

I could fix that, though.

Reply:
Let's just drop the whole subclass thing and have one class that takes the schema name